src/patternfly/components/Wizard/wizard.scss

```diff
@@ -393,11 +393,9 @@
   background-color: var(--pf-c-wizard__nav--BackgroundColor);
   box-shadow: var(--pf-c-wizard__nav--BoxShadow);
 
-  @media screen and (max-width: $pf-global--breakpoint--lg) {
-    &.pf-m-expanded {
-      display: block;
-      visibility: visible;
-    }
+  &.pf-m-expanded {
+    display: block;
+    visibility: visible;
```

Comment:
This looks good, but I'm seeing remnants of previous commits in the preview CSS. For example, the nav isn't visible in this demo https://patternfly-next-pr-2816.surge.sh/documentation/core/components/wizard/nav-expanded-mobile

It's due to this block of CSS being in the compiled CSS on the preview site, but not in our SCSS from as far as I can tell. I think it _was_ in our SCSS, but has since been removed?

```
@media screen and (min-width: 992px) {
    .pf-c-wizard__nav.pf-m-expanded {
        display:none;
        visibility: hidden
    }
}
```

@redallen can you check this out? I tried re-running the build and still seeing this. I left a similar comment here - https://github.com/patternfly/patternfly/pull/2816#issuecomment-603531287